CL/cl_ext.h

```diff
@@ -2347,6 +2347,19 @@ clEnqueueMemsetINTEL(
 
 #endif /* CL_NO_PROTOTYPES */
 
+/***************************************************************
+* cl_mem_alloc_buffer_location_intel
```

Comment:
Extension name is `cl_intel_mem_alloc_buffer_location` not `cl_mem_alloc_buffer_location_intel`  as these defines suggest